src/ResourceManager/ApplicationInsights/Commands.ApplicationInsights/Get-AzureRmApplicationInsights.md

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
@frankguodongchen reviewed this file and noticed that it wasn't in the `help` folder like all other markdown files. Any reason for that?

Reply:
looks like extra file accidently checked in, removed, thanks!